src/tabs/presets/DetailedDialog/PresetsDetailedDialog.css

```diff
@@ -73,6 +73,10 @@
     display: inline-block;
 }
 
+.ms-drop ul>li.hide-radio:focus, .ms-drop ul>li.hide-radio:hover {
```

Comment:
could you please add something like:
```
#presets_detailed_dialog
.ms-drop ul>li.hide-radio:focus, .ms-drop ul>li.hide-radio:hover {
```

so that this style has been applied only for this particular dialog.

Reply:
Umm, really needed? Its already on presets_detailed_dialog css file so only works here.

Reply:
this css file is included from main html, so all the styles from this file will be applied to the whole Configurator. Unless you narrow them with more specific selectors.

Reply:
Ouch! Understood, thought that it is included amd comes only in presets dialog html and js files.